docs/publish/centralized-deployment.md

```diff
@@ -45,38 +45,37 @@ If your organization meets all requirements, complete the following steps to pub
 
 1. Sign in to Office 365 with your work or school account.
 2. Select the app launcher icon in the upper-left and choose **Admin**.
-3. In the navigation menu, choose **Settings** > **Services & add-ins**.
+3. In the navigation menu, press **Show more**, then choose **Settings** > **Services & add-ins**.
 4. If you see a message on the top of the page announcing the new Office 365 admin center, choose the message to go to the Admin Center Preview (see [About the Office 365 admin center](https://support.office.com/en-ie/article/About-the-Office-365-admin-center-758befc4-0888-4009-9f14-0d147402fd23)).
-5. Choose **Upload Add-in** at the top of the page. 
-6. Choose one of the following options on the **Centralized Deployment** page:
+5. Choose **Deploy Add-In** at the top of the page. 
+6. Choose *Next** after reviewing the requirements.
+7. Choose one of the following options on the **Centralized Deployment** page:
 
-    - **I want to add an add-in from AppSource.**
+    - **I want to add an Add-In from the Office Store.**
     - **I have the manifest file (.xml) on this device.** For this option, choose **Browse** to locate the manifest file (.xml) that you want to use.
     - **I have a URL for the manifest file.** For this option, type the manifest's URL in the field provided.
 
     ![New Add-In dialog in Office 365 admin center](../images/new-add-in.png)
 
-7.	Choose **Next**.
-
-8.	If you selected the option to add an add-in from AppSource, select the add-in. Notice that you can view available add-ins via categories of **Suggested for you**, **Rating**, or **Name**. You may only add free add-ins from AppSource; adding paid add-ins isn't currently supported.
+8.	If you selected the option to add an add-in from Office Store, select the add-in. You can view available add-ins via categories of **Suggested for you**, **Rating**, or **Name**. You may only add free add-ins from Office Store. Adding paid add-ins isn't currently supported.
 
     > [!NOTE]
-    > With the AppSource option, updates and enhancements to the add-in will automatically be made available to users without your intervention.
+    > With the Office Store option, updates, and enhancements to the add-in are automatically available to users without your intervention.
```

Comment:
remove the comma after "updates"  (i.e., should read:  "updates and enhancements)